frontend/src/components/AccountPicker.vue

```diff
@@ -41,9 +50,14 @@ const isXlScreen = useMedia('(min-width: 1280px)');
         }}</span>
       </span>
     </span>
+    <span class="account-picker-content" v-else-if="!isMetaMaskInstalled">
+      <span>
+        <span>Install MetaMask</span>
```

Comment:
Ideally this should be shown immediately in place of the "Connect Wallet" button, if Metamask is not installed.